spec/requests/api/v1/invoice_request_relationship_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
+describe "Invoice API" do
+  it "returns a collection of associated transactions" do
+    invoice = Fabricate(:invoice)
+    Fabricate.times(2, :transaction, invoice: invoice)
+    get "/api/v1/invoices/#{invoice.id}/transactions"
+
+    expect(response).to be_success
+    transactions = JSON.parse(response.body)
+    expect(transactions.count).to eq(2)
+    expect(transactions.class).to eq(Array)
+    expect(transactions.first["invoice_id"]).to eq(invoice.id)
+    expect(transactions.last["invoice_id"]).to eq(invoice.id)
```

Comment:
This test could also check that transactions not associated with that invoice_id are not returned. This happens to return the correct value but doesn't check that unrelated transactions are not returned.